src/OpalCompiler-Tests/OCBytecodeToASTCacheTest.class.st

```diff
@@ -73,5 +73,5 @@ OCBytecodeToASTCacheTest >> testNodeForBCOffsetTest [
 	| pc |
 	"there should be an assignment at pc 51"
 	pc := 51.
-	self assert: (cache nodeForPC: pc) isAssignment
+	self assert: (cache nodeForPC: pc) sourceCode equals: 'i := i - 1'.
```

Comment:
I would also check:
- node class
- node start
- node stop
This way we're sure it is the correct one, as source code strings can be printed from different nodes.

Reply:
There is just one expression "i := i - 1" in th method under test. there is no way that we print that string and it is not the node we are looking for

Reply:
But I'm concerned by the fact that we may change the test method one day, and introduce such subtility that would make the test pass while it should not.
As we're testing the node mapping, I think we need to always make sure that we precisely identify the node we test.

Reply:
ok, added stop and start check